GCM/App/userInfo/UserInfo.java

```diff
@@ -1,7 +1,11 @@
 package userInfo;
 
+import users.User;
+
 public interface UserInfo {
 
 	boolean login(String username, String password);
+	
+	boolean register(String username, String password,User user);
```

Comment:
```suggestion
	/**
	* Executes a registration request.
	* @return true if user was registered successfully, false otherwise
	*/
	boolean register(String username, String password,User user);
```